build/extract-specifications.ts

```diff
@@ -71,7 +71,8 @@ export function extractSpecifications(
     }
   }
 
-  if (query) {
+  // Look for spec URLS in BCD only if specURLsString is not set
```

Comment:
Let's focus more on the intention:

```suggestion
  // The `spec-url` frontmatter takes precedence over spec URLs derived from `browser-compat` frontmatter.
```